OpenChange/RTFHandler.m

```diff
@@ -529,109 +763,137 @@ - (RTFColorTable *) parseColorTable
 //
 // We receive the full string.
 //
+
+
 - (RTFFontTable *) parseFontTable
 {
```

Comment:
Neither is a 'parse standard', nor we have a event-driven parser and this is not  a library for third-parties. Please, focus on our product.